pootle/apps/pootle_store/models.py

```diff
@@ -1294,14 +1288,10 @@ def toggle_qualitycheck(self, check_id, false_positive, user):
             sub_type = SubmissionTypes.UNMUTE_CHECK
 
         sub = Submission(creation_time=timezone.now(),
-            translation_project=self.store.translation_project,
-            submitter=user,
-            field=SubmissionFields.NONE,
-            unit=self,
-            store=self.store,
-            type=sub_type,
-            quality_check=check
-        )
+                         translation_project=self.store.translation_project,
+                         submitter=user, field=SubmissionFields.NONE,
+                         unit=self, store=self.store, type=sub_type,
+                         quality_check=check)
```

Comment:
Previous code allowed to easily spot each argument on its own line. Is it not possible to keep it that way by just appending a comma after the last argument?

The same goes for several other similar changes, but would like to first hear other opinions about this particular instance.